### PR TITLE
added redirect

### PR DESCRIPTION
**Pull request by joel-hamill**
CC: @rcorral

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,5 @@ build
 npm-debug.log
 node_modules
 .idea
+dd
+.gitignore
```
Comment (karlkfi) [inline .gitignore]: this is wrong. you dont want to ingore gitignore...

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,5 @@ build
 npm-debug.log
 node_modules
 .idea
+dd
```
Comment (karlkfi) [inline .gitignore]: why was this added?

Comment (joel-hamill) [inline .gitignore]: @karlkfi this is why i tried to ignore .gitignore. i have no idea what `+dd` is.

Comment (karlkfi) [inline .gitignore]: just remove it from your local repo then
